tests/test_template_init.py

```diff
@@ -84,26 +76,53 @@ def answers() -> dict[str, str]:
     }
 
 
+@pytest.fixture
+def generated(tmp_path: Path, answers: dict[str, str]) -> Callable[..., Path]:
+    """Fixture closure to generate a template project, overriding passed data values."""
+    def _generated(**kwargs) -> Path:
+        out_path = tmp_path
```

Comment:
Seems to have no effect, why not get rid of it and rename in following lines?

```suggestion
```

Reply:
aha i wasn't sure if it would be determinstic within a test invocation, but now that i think of it how could it possibly not be that's the whole point of what it does